cmd/server/reader.go

```diff
@@ -8,40 +8,47 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strings"
 
 	"github.com/moov-io/base/log"
 	"github.com/moov-io/fed"
 	"github.com/moov-io/fed/pkg/download"
 )
 
 func fedACHDataFile(logger log.Logger) (io.Reader, error) {
-	if file, err := attemptFileDownload(logger, "fedach"); file != nil {
-		return file, nil
-	} else if err != nil {
+	file, err := attemptFileDownload(logger, "fedach")
+	if err != nil && !strings.Contains(err.Error(), download.MissingRoutingNumberErr) && !strings.Contains(err.Error(), download.MissingDownloadCodeErr) {
```

Comment:
This is a bit tough to read.  Maybe use an error type as suggested by @adamdecaf below, or use a method, e.g.
```
if isMissingDataError(err) {
    // ...
}

func isMissingDataError(err) {
    return err != nil && !strings.Contains(err.Error(), download.MissingRoutingNumberErr) && !strings.Contains(err.Error(), download.MissingDownloadCodeErr)
}
```